stdlib/builtin/kernel.rbs

```diff
@@ -333,13 +313,14 @@ module Kernel
   # `Exception` object or `nil`, can be specified via the `:cause`
   # argument.
   def fail: () -> bot
-          | (?String arg0) -> bot
-          | (?Class arg0, ?::Array[String] arg1) -> bot
-          | (?Class arg0, ?String arg1, ?::Array[String] arg2) -> bot
+          | (String arg0) -> bot
+          | (?Exception arg0, ?String arg1, ?::Array[String] arg2) -> bot
```

Comment:
I think the type of `fail` should be updated later, but not clear enough how it can be now.